src/stream.ts

```diff
@@ -108,6 +108,7 @@ export interface Constraints extends MediaStreamConstraints {
   resolution: string;
   codec: string;
   simulcast?: boolean;
+  dummyTracks?: boolean;
```

Comment:
```suggestion
  dummyTracks?: boolean;
```
we could improve the naming here. perhaps `sendEmptyOnMute`